ansible/roles/postgresql/tasks/main.yml

```diff
@@ -90,16 +90,7 @@
   tags: postgresql_conf
   notify: Reload postgres
 
-- name: Restart postgresql
-  become: true
-  service: name=postgresql state=restarted args="{{ postgresql_version }}"
-  tags: after-reboot
-  register: start_postgresql
-
-- name: wait after restarting postgres
-  when: start_postgresql.changed
-  wait_for:
-    port: '{{ postgresql_port }}'
+- meta: flush_handlers
```

Comment:
should you keep the wait?

Reply:
I tested it and I didn't have to, so I figured I'd remove and add back if it's ever a problem. I originally added it because I misunderstood what the problem was and thought it would help (it didn't).